bastion.cfn.yml

```diff
@@ -16,68 +16,290 @@ Parameters:
 
   KeyName:
     Description: EC2 key pair name for bastion host SSH access
+    Type: AWS::EC2::KeyPair::KeyName
+
+  LogRetentionInDays:
+    Description: Number of days you would like you cloudwatch logs to be retained
+    Type: Number
+    Default: 7
+
+  MFA:
+    Description: Change it to true to install MFA using Google Authenticator on your instance
```

Comment:
Can this be more descriptive? I may not know what Google Authenticator is - is there a link where I can read up?

Reply:
We will work with John M. on adding documentation. He is working on major overhaul of the main docs.